docs/reference/configuration/Storage.md

```diff
@@ -64,10 +64,9 @@ Name: littlefs.enable_warn
     No value set
 ```
 
-
 ### NVStore configuration
 
-NVStore does not need much configuration, relying only on the regions of internal flash specified in the `area_*_address` and `area_*_size` for the two areas. Additionally, `max_keys` can be used to manage the amount of RAM needed for NVStore keys. Note that `max_keys` will default to the number of keys needed by Mbed OS and only needs to be modified if an application uses NVStore directly.
+NVStore does not need much configuration. It relies only on the regions of internal flash specified in the `area_*_address` and `area_*_size` for the two areas. Additionally, you cna use `max_keys` to manage the amount of RAM NVStore keys needs. Note that `max_keys` defaults to the number of keys Mbed OS needs. You only need to be modify it if an application uses NVStore directly.
```

Comment:
Typo: "can" instead of "cna"
Typo: "You only need to modify it" instead of "You only need to be modify it"

Reply:
This is why everyone needs an editor. Thank you for catching these.

Reply:
Oh wow, what moron wrote this?

Reply:
Sorry, that comment was very inappropriate. I thought it was my own writing that was being corrected and tried to make a joke.

Either way, it wasn't an acceptable comment to write. @AnotherButler thanks for all the help editting these docs, I'm sure I haven't made it easy for you to catch all of the writing mistakes.

Reply:
@geky No worries. It's always a pleasure editing your docs.